### PR TITLE
Improve the support for pseudo-elements

### PR DESCRIPTION
**Pull request by esdoppio**
pls merge this after #85 😉

### CODE REVIEW

**Comment by 50Wliu** [comment]
Like before, please rebase.

**Comment by esdoppio** [comment]
Ready to 🎸

File: grammars/css.cson [hunk 1]
```diff
@@ -556,14 +556,22 @@
         'captures':
           '1':
             'name': 'punctuation.definition.entity.css'
-        'match': '(:+)(after|before|content|first-letter|first-line|host|(-(moz|webkit|ms)-)?selection)\\b'
+          '2':
+            'name': 'punctuation.definition.entity.css'
+        'match': '''(?x)
+          (?:
+            (::?+)(?:after|before|first-letter|first-line|(?:-(?:webkit|moz|ms)-)[a-z-]++)
```
Comment (50Wliu) [inline grammars/css.cson]: Why are we doing possessive matching on the `::` and the `[a-z-]`?

Comment (esdoppio) [inline grammars/css.cson]: To avoid meaningless backtracks.

**Pull request merged**